cmd/stack_package.go

```diff
@@ -611,3 +618,13 @@ func ApplyTemplating(stackPath string, templateMetadata interface{}) error {
 	return nil
 
 }
+
+// stringInSlice is used to check for allowed application files
+func stringInSlice(a string, list []string) bool {
```

Comment:
I think we could rename this function to `contains`, because in the end we are just checking that a string is in an array of strings. A def example would be `contains(array, value)`.